client/components/automattic-logo/index.tsx

```diff
@@ -1,15 +1,19 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { FunctionComponent } from 'react';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const AutomatticLogo = ( { className, size } ) => (
+interface Props {
+	className?: string;
+	size?: number;
+}
+
+const AutomatticLogo: FunctionComponent< Props > = ( { className = '', size = 96 } ) => (
 	<svg className={ `automattic-logo ${ className }` } height={ size } viewBox="0 0 792 96">
```

Comment:
I don't like that with the default we wind up with class `automattic-logo ` (note trailing space). How about something like this:

```suggestion
	<svg className={ 'automattic-logo' + ( className ? className : '' ) } height={ size } viewBox="0 0 792 96">
```

Reply:
But now `'automattic-logo'` and the `className` are no longer separated by a space at all, are they?

Reply:
Good old

```suggestion
	<svg className={ compact( [ 'automattic-logo', className ] ).join( ' ' ) } height={ size } viewBox="0 0 792 96">
```

maybe?

Reply:
Or use the `classNames` package? :grimacing:

Reply:
5dd5366